docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -311,59 +311,52 @@ Default: `64`
 
 The number of shards per YB-TServer for each YCQL table when a user table is created.
 
-Default: `-1` (the value is calculated at runtime). For servers with up to two CPU cores, the default value is considered as `4`. For three or more CPU cores, the default value is considered as `8`. Local cluster installations, created with `yb-ctl` and `yb-docker-ctl`, use a value of `2` for this flag. Clusters created with `yugabyted` use a default value of `1`.
+Default: `-1` (the value based on the CPU cores is calculated at runtime).
+\
+For servers with up to two CPU cores, the default value is considered as `4`.
+\
+For three or more CPU cores, the default value is considered as `8`.
+\
+If [`enable_automatic_tablet_splitting`](#enable-automatic-tablet-splitting) is `true`, then the default value is considered as `1` and tables will begin with 1 tablet *per node*; for version `2.18` and later, for servers with up to 4 CPU cores, the value *is not defined* and tables will begin with 1 tablet (for servers with up to 2 CPU cores), or 2 tablets (for servers with up to 4 CPU cores) *per cluster*.
 
-{{< note title="Important" >}}
+**For clusters installed with `yb-ctl`, `yb-docker-ctl`, and `yugabyted`**
```

Comment:
I would remove this header as it looks like a Note section belongs to this particular section.